lib/index.js

```diff
@@ -90,8 +79,6 @@ export {
 
   // mobile components
   MobileMain,
-  MobileResultsScreen,
```

Comment:
I'm getting an error when trying to run call taker. it seems to want to import this for its mobile view. This can probably be replaced with something else but I think I'd prefer to not have to update all the mobile configs. I think the solution here is a "fake" export in the right place. 

If you like we can discuss more offline

Reply:
Replacing this with fake exports seems to work ok, although it seems to crash when I try to actually plan a trip. (it crashes the tab, so no error message.)

Reply:
I wonder what the problem is? Do you think it might be that the call taker mobile views are somehow broken?